hack/generate-tests.sh

```diff
@@ -49,7 +49,7 @@ startKIND: true
 kindNodeCache: true
 kindContainers:
   - storageos/operator:test
-timeout: 800
+timeout: 643
```

Comment:
that's very specific, any reason behind 643?

Reply:
It is a prime number :) 800 is too much, 500 is too low. 643 is somewhere in the middle.